packages/http-client-csharp/generator/Microsoft.TypeSpec.Generator/src/CodeModelGenerator.cs

```diff
@@ -132,5 +134,11 @@ public void AddTypeToKeep(string typeName)
         /// </summary>
         /// <param name="type">The type provider representing the type.</param>
         public void AddTypeToKeep(TypeProvider type) => AddTypeToKeep(type.Type.FullyQualifiedName);
+
+        /// <summary>
+        /// Adds a type to the list of types to keep as public.
+        /// </summary>
+        /// <param name="type">The type provider representing the type.</param>
+        public void AddTypeToKeepPublic(string type) => TypesToKeepPublic.Add(type);
```

Comment:
How is this different than the other overload that takes a string?

Reply:
Oh nevermind this is type to keep public. Can we rename the parameter to match the convention?